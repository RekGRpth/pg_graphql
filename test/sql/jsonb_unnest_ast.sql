with doc(x) as (
    select (graphql.parse(
        $$
            {
              accountConnection(last: 2, before: "WyJhY2NvdW50IiwgM10=") {
                edges {
                  node {
                    nodeId
                    id
                  }
                }
              }
            }
        $$
    )).ast
)
select
    jkr.jpath,
    jkr.obj
from
    doc,
    graphql.jsonb_unnest_recursive_with_jsonpath(graphql.ast_pass_strip_loc(doc.x::jsonb)) jkr(jpath, obj)
where
    jkr.jpath::text not ilike '%."loc"%'
