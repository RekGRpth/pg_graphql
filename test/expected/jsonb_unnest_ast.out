with doc(x) as (
    select (graphql.parse(
        $$
            {
              accountConnection(last: 2, before: "WyJhY2NvdW50IiwgM10=") {
                edges {
                  node {
                    nodeId
                    id
                  }
                }
              }
            }
        $$
    )).ast
)
select
    jkr.jpath,
    jkr.obj
from
    doc,
    graphql.jsonb_unnest_recursive_with_jsonpath(graphql.ast_pass_strip_loc(doc.x::jsonb)) jkr(jpath, obj)
where
    jkr.jpath::text not ilike '%."loc"%'
                                                                             jpath                                                                             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        obj                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
---------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 $                                                                                                                                                             | {"kind": "Document", "definitions": [{"kind": "OperationDefinition", "name": null, "operation": "query", "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}]}, "variableDefinitions": null}]}
 $."definitions"                                                                                                                                               | [{"kind": "OperationDefinition", "name": null, "operation": "query", "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}]}, "variableDefinitions": null}]
 $."definitions"[0]                                                                                                                                            | {"kind": "OperationDefinition", "name": null, "operation": "query", "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}]}, "variableDefinitions": null}
 $."definitions"[0]."directives"                                                                                                                               | null
 $."definitions"[0]."kind"                                                                                                                                     | "OperationDefinition"
 $."definitions"[0]."name"                                                                                                                                     | null
 $."definitions"[0]."operation"                                                                                                                                | "query"
 $."definitions"[0]."selectionSet"                                                                                                                             | {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}]}
 $."definitions"[0]."selectionSet"."kind"                                                                                                                      | "SelectionSet"
 $."definitions"[0]."selectionSet"."selections"                                                                                                                | [{"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}]
 $."definitions"[0]."selectionSet"."selections"[0]                                                                                                             | {"kind": "Field", "name": {"kind": "Name", "value": "accountConnection"}, "alias": null, "arguments": [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}], "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}}
 $."definitions"[0]."selectionSet"."selections"[0]."alias"                                                                                                     | null
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"                                                                                                 | [{"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}, {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}]
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]                                                                                              | {"kind": "Argument", "name": {"kind": "Name", "value": "last"}, "value": {"kind": "IntValue", "value": "2"}}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."kind"                                                                                       | "Argument"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."name"                                                                                       | {"kind": "Name", "value": "last"}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."name"."kind"                                                                                | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."name"."value"                                                                               | "last"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."value"                                                                                      | {"kind": "IntValue", "value": "2"}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."value"."kind"                                                                               | "IntValue"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[0]."value"."value"                                                                              | "2"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]                                                                                              | {"kind": "Argument", "name": {"kind": "Name", "value": "before"}, "value": {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."kind"                                                                                       | "Argument"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."name"                                                                                       | {"kind": "Name", "value": "before"}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."name"."kind"                                                                                | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."name"."value"                                                                               | "before"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."value"                                                                                      | {"kind": "StringValue", "value": "WyJhY2NvdW50IiwgM10="}
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."value"."kind"                                                                               | "StringValue"
 $."definitions"[0]."selectionSet"."selections"[0]."arguments"[1]."value"."value"                                                                              | "WyJhY2NvdW50IiwgM10="
 $."definitions"[0]."selectionSet"."selections"[0]."directives"                                                                                                | null
 $."definitions"[0]."selectionSet"."selections"[0]."kind"                                                                                                      | "Field"
 $."definitions"[0]."selectionSet"."selections"[0]."name"                                                                                                      | {"kind": "Name", "value": "accountConnection"}
 $."definitions"[0]."selectionSet"."selections"[0]."name"."kind"                                                                                               | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."name"."value"                                                                                              | "accountConnection"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"                                                                                              | {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."kind"                                                                                       | "SelectionSet"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"                                                                                 | [{"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}]
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]                                                                              | {"kind": "Field", "name": {"kind": "Name", "value": "edges"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."alias"                                                                      | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."arguments"                                                                  | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."directives"                                                                 | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."kind"                                                                       | "Field"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"                                                                       | {"kind": "Name", "value": "edges"}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."kind"                                                                | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."value"                                                               | "edges"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"                                                               | {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."kind"                                                        | "SelectionSet"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"                                                  | [{"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}]
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]                                               | {"kind": "Field", "name": {"kind": "Name", "value": "node"}, "alias": null, "arguments": null, "directives": null, "selectionSet": {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."alias"                                       | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."arguments"                                   | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."directives"                                  | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."kind"                                        | "Field"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"                                        | {"kind": "Name", "value": "node"}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."kind"                                 | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."value"                                | "node"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"                                | {"kind": "SelectionSet", "selections": [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."kind"                         | "SelectionSet"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"                   | [{"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}, {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}]
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]                | {"kind": "Field", "name": {"kind": "Name", "value": "nodeId"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."alias"        | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."arguments"    | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."directives"   | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."kind"         | "Field"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"         | {"kind": "Name", "value": "nodeId"}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."kind"  | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."name"."value" | "nodeId"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet" | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]                | {"kind": "Field", "name": {"kind": "Name", "value": "id"}, "alias": null, "arguments": null, "directives": null, "selectionSet": null}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."alias"        | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."arguments"    | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."directives"   | null
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."kind"         | "Field"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."name"         | {"kind": "Name", "value": "id"}
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."name"."kind"  | "Name"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."name"."value" | "id"
 $."definitions"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[0]."selectionSet"."selections"[1]."selectionSet" | null
 $."definitions"[0]."variableDefinitions"                                                                                                                      | null
 $."kind"                                                                                                                                                      | "Document"
(79 rows)

